src/ConsoleMain/Main.cpp

```diff
@@ -15,6 +15,7 @@ namespace
     const std::string s_defaultOutputPathFileCompilations = "FileCompilations.csv";
     const std::string s_defaultOutputPathBuildTimeline = "BuildTimeline.json";
     const std::string s_defaultOutputPathTemplateInstantiations = "TemplateInstantiations.csv";
+    const std::string s_defaultTimingDisplay = "ns";
```

Comment:
I really like this new option, and I really feel like the default should be seconds, which are easier to reason about.
What do you think?

Reply:
Uhhmm. I have mixed feelings about this.
I for one, might be biased, but i reason better in milliseconds.
I don't think nanoseconds are very good because the numbers are too big but on the other hand for seconds the numbers are too small.
Also notice that I didn't modify the template analyzer for this very reasons. That there it makes more sense to use nanoseconds.

Reply:
Let's keep nanoseconds as the default then 👍